playbook/app/pb_kits/playbook/utilities/icons/allicons.tsx

```diff
@@ -174,7 +175,8 @@ export const getAllIcons = () => {
             icon: arrowDownShortWide
         },
         angleDown: {
-            icon: angleDown
+            icon: angleDown,
+            string: ReactDOMServer.renderToStaticMarkup(angleDown)
```

Comment:
flatpikr needs a string to "un-safely" pass as an arg so the calendar has a down arrow

Reply:
Why are we adding all these icons manually as SVGs directly? Is there a way we can use the playbook-icons library?

Reply:
Flatpkr uses fa icon so i stringified the svg.

 playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.ts

```
dropdown.insertAdjacentHTML('afterend', '<i class="far fa-angle-down year-dropdown-icon" id="test-id"></i>')
```

Reply:
After talking with Mark more, I see where we are headed with this for now.